xla/hlo/evaluator/hlo_evaluator.cc

```diff
@@ -1696,7 +1701,7 @@ absl::Status HloEvaluator::HandleTuple(const HloInstruction* tuple) {
     CHECK(new_result.IsDetermined(visitor_shape_index_));
     Literal literal;
     TF_RETURN_IF_ERROR(
-        literal.CopyFrom(std::move(new_result),
+        literal.CopyFrom(new_result,
```

Comment:
Literal::CopyFrom expects a reference (&), not an r-value reference (&&). Removing std::move